week-4/Homework/mandatory/2-writers.js

```diff
@@ -42,3 +42,11 @@ let writers = [
 /*
 If you want an extra challenge, only `console.log()` the writers that are alive.
 */
+
+//writers.forEach(writer => console.log( `Hi, my name is ${writer.firstName} ${writer.lastName}. I am ${writer.age} years old, and work as a ${writer.occupation}.`));
+
+writers.filter(writer => writer.alive === true).forEach(writer => console.log(
+  `Hi, my name is ${writer.firstName} ${writer.lastName}. I am ${writer.age} years old, and work as a ${writer.occupation}.`));
+

```

Comment:
You can use like (writer=>writer.alive) it means same as you used (writer => writer.alive === true). Good work well done:)

Reply:
Thank you